docker-for-windows/index.md

```diff
@@ -46,6 +46,8 @@ docker-machine version 0.10.0, build 76ed2a6
 
 The next few steps take you through some examples. These are just suggestions for ways to experiment with Docker on your system, check version information, and make sure `docker` commands are working properly.
 
+**Note for Windows Users**: Remember to use double quotes (") instead of single quotes (') when specifying parameters in docker commands while using CMD/Powershell.
```

Comment:
Please make this an actual Markdown note and title it something different. For instance:

```markdown
> Powershell syntax
>
> Remember to use....
```